com.thebrokenrail.MCPIReborn.yml

```diff
@@ -0,0 +1,193 @@
+app-id: com.thebrokenrail.MCPIReborn
+
+# MCPI Itself Is Proprietary
+tags:
+  - proprietary
+
+# Runtime/SDK
+runtime: org.freedesktop.Platform
+# Newer Runtimes Don't Support ARM Multiarch
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+
+# Startup Command
+command: minecraft-pi-reborn-client
+
+# Permissions
+finish-args:
+  - --persist=.minecraft-pi
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
does this support wayland?

Reply:
GLFW does if specifically compiled, but not by default. I'll just remove it.

Reply:
I removed it.